mscore/dragelement.cpp

```diff
@@ -98,7 +98,8 @@ void ScoreView::doDragElement(QMouseEvent* ev)
 
       for (Element* e : sel.elements()) {
             QVector<QLineF> elAnchorLines = e->dragAnchorLines();
-            const QPointF pageOffset(e->findAncestor(ElementType::PAGE)->pos());
+            Element* el = e->findAncestor(ElementType::PAGE);
+            const QPointF pageOffset(el ? el->pos() : QPointF());
```

Comment:
Change to:

`const QPointF pageOffset((page ? page : e)->pos());`

Reply:
nope, doesn't even compile. But `const QPointF pageOffset((page() ? page() : e)->pos());` does ;-)

Reply:
And doing it this way indeed seems way better, thanks!